src/components/MessageInput/hooks/useAttachments.ts

```diff
@@ -89,25 +105,43 @@ export const useAttachments = <
     [maxFilesLeft, noFiles],
   );
 
-  const removeAttachment = (id: string) => {
-    dispatch({ id, type: 'removeAttachment' });
-  };
+  const removeAttachments = useCallback(
+    (ids: string[]) => {
+      if (!(ids && ids.length)) return;
```

Comment:
```suggestion
      if (!ids.length) return;
```
According to the type annotation `ids` should always be an array so this simple check should suffice.

Reply:
I was too paranoid :)

Reply:
adjusted